cloud-controller-manager/do/loadbalancers.go

```diff
@@ -793,6 +805,8 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 		return nil, err
 	}
 
+	// we don't pass UDP ports here since we can allow duplicate
```

Comment:
Ugh, while LB supports listening on the same port for both udp and tcp looks like Kubernetes does not:
https://github.com/kubernetes/kubernetes/issues/39188
https://github.com/kubernetes/kubernetes/issues/2995#issuecomment-267024643

I guess I'll rework this to be consistent with the other ports to avoid confusion for users.